bundle-size/package.json

```diff
@@ -25,7 +25,8 @@
     "node-cache": "5.1.0",
     "pg": "7.14.0",
     "probot": "9.6.6",
-    "sleep-promise": "8.0.1"
+    "sleep-promise": "8.0.1",
+    "travis-ips": "file:../shared/travis-ips"
```

Comment:
I'm curious whether this file would get pushed to the cloud instance given that it's a level _above_

Reply:
Interesting, I see what you mean. I'm not sure how to test it
I think the GCloud Build approach would work fine (since the whole repo is present on the build system, not just the app folder) but I bet this could fail with `gcloud app deploy`...